bin/wp

```diff
@@ -12,8 +12,4 @@ CLI="wp $COMMAND"
 # Mimic core `wp-cli`'s behavior of using `less` for `help` output.
 PAGER=$(if [ -z "$COMMAND" ] || [[ "$COMMAND" == \'help* ]]; then echo "less -R"; else echo "cat"; fi)
 
-if [ -n "$STUDIO_APP_DIRECTORY" ] && [ -n "$STUDIO_EXE_PATH" ]; then
-  "$STUDIO_EXE_PATH" "$STUDIO_APP_DIRECTORY" --cli="$CLI" | $PAGER
-else
-  "$STUDIO_APP_PATH" --cli="$CLI" | $PAGER
-fi
+$STUDIO_APP_PATH --cli="$CLI" | $PAGER
```

Comment:
TIL. I realized that my solution was a bit hacky, but couldn't make it work! Thanks!